README.md

```diff
@@ -2458,6 +2458,179 @@ You can query multiple elements, for example you can add the section with the im
 ```
  <Debugging query="product=Photoshop&image=test" />
 ```
+# Get Credential 
+
+The Get Credential component allows you to easily generate credentials and access code samples directly from the developer website. Child component should only work if it's inside the parent GetCredential component 
+
+Use `credentialType`(optional) prop to specify the credential type . Defaults to `apiKey`.
+
+Use `service`(optional) prop to define the Adobe Product & Service.The preset is `CCEmbedCompanionAPI`.
+
+Use `className` to customize the component **at your own risk.**.
+
+### GetCredential.SignIn
+
+`GetCredential.SignIn` is a component utilized to prominently showcase the sign-in page, offering users a straightforward access point to login.
+
+Use `title`(required)  to provide a title for the credential.
+
+Use `paragraph`(optional) to describe the credentials.
+
+Use `buttonText`(required) to specify the sign-in button label
+
+`GetCredential.Form` is a flexible for designing the credential form, ensuring a seamless and personalized user experience.
+
```

Comment:
Can you add a header for each one of the different headings?  Just like the GetCredential.SignIn? It's easier to read if we have different headings.